proposals/description/tip_123.ts

```diff
@@ -0,0 +1,138 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+import { ethers } from 'ethers';
+
+const tip_123: TemplatedProposalDescription = {
+  title: 'TIP_123',
+  commands: [
+    // 1. Transfer beneficiary of deprecated Rari FEI timelock to burner timelock
+    {
+      target: 'rariInfraFeiTimelock',
+      values: '0',
+      method: 'setPendingBeneficiary(address)',
+      arguments: (addresses) => [addresses.deprecatedRariFeiTimelockBurner],
+      description: 'Set pending beneficiary of deprecated Rari Fei timelock burner to Fei burner timelock'
+    },
+    {
+      target: 'deprecatedRariFeiTimelockBurner',
+      values: '0',
+      method: 'acceptBeneficiary()',
+      arguments: (addresses) => [],
+      description: 'Accept deprecated Rari Fei timelock beneficiary to burner'
+    },
+
+    // 2. Transfer beneficiary of deprecated Rari TRIBE timelock to burner timelock
+    {
```

Comment:
The one side effect here is that the TRIBE will remain delegated to Rari Infra multisig as it unlocks. Probably best to do a "delegate(address)(0x0) call" beforehand

Reply:
Done